EIPS/eip-x.md

```diff
@@ -54,11 +60,11 @@ After submission, Requests go through Pending, Claimable, and Claimed stages. An
 | Claimable |                                  | <i>Internal request fulfillment</i><br>pendingDepositRequest[msg.sender] -= assets<br>maxDeposit[operator] += assets<br> |
 |   Claimed | deposit(assets, receiver)        |                                                  maxDeposit[msg.sender] -= assets<br>vault.balanceOf[receiver] += shares |
 
-The requested amount is defined by `pendingDepositRequest + maxDeposit`.
+An important vault invariant is that following a request(s), the cumulative requested quantity MUST equal `pendingDepositRequest + maxDeposit - claimed`. The previous invariant assumes there are no cancel requests, whose behavior is discussed below.
```

Comment:
if requests are cancelled, wouldn't `maxDeposit` go up the same amount as `pendingDepositRequest` goes down? invariant of `pendingDepositRequest + maxDeposit` should still hold right. might be missing something with `claimed`

Reply:
It should still hold as an inequality but pendingDeposit may go down before maxDeposit recovers